src/store/reducers/todo.js

```diff
@@ -36,6 +37,16 @@ export const todoReducer = (state = initialTodoState, action) => {
       return { ...state, tasks: existingTasks };
     }
 
+    case EDIT_TASK: {
+      const existingTasks = [...state.tasks];
+      const selectedTaskId = findTaskIndexById(
+        action.payload.taskId,
+        existingTasks
+      );
+      existingTasks[selectedTaskId].title = action.payload.title;
```

Comment:
// see https://eslint.org/docs/latest/rules/prefer-destructuring

```suggestion
      const { taskId, title } = action.payload
      const selectedTaskId = findTaskIndexById(taskId, existingTasks);
      existingTasks[selectedTaskId].title = title;
```